storage_controller/src/http.rs

```diff
@@ -1174,6 +1186,13 @@ pub fn make_router(
                 RequestName("control_v1_tenant_policy"),
             )
         })
+        .put("/control/v1/preferred_azs", |r| {
```

Comment:
Nice -- we may well use this in the field when we see tenants that have long running computes in the "wrong" AZ from our earlier az-naive scheduling decisions.